api/tools/src/main/java/org/pmiops/workbench/tools/ImpersonatedServiceAccountApiClientFactory.java

```diff
@@ -0,0 +1,43 @@
+package org.pmiops.workbench.tools;
+
+import com.google.cloud.iam.credentials.v1.IamCredentialsClient;
+import com.google.protobuf.Duration;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.firecloud.ApiClient;
+
+public class ImpersonatedServiceAccountApiClientFactory extends ApiClientFactory {
+
+  public ImpersonatedServiceAccountApiClientFactory(
+      String targetServiceAccount, WorkbenchConfig config) {
+    try {
+      this.apiClient = newApiClient(targetServiceAccount, config);
+    } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
It doesn't seem correct to allow error suppression here - would expect a runtime error to be propagated